src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestStage.java

```diff
@@ -246,7 +245,7 @@ synchronized SnowflakeFileTransferMetadataWithAge refreshSnowflakeMetadata(boole
 
     Map<Object, Object> payload = new HashMap<>();
     payload.put("role", this.role);
-    Map<String, Object> response = this.makeClientConfigureCall(payload);
+    ConfigureResponse response = this.makeClientConfigureCall(payload);
```

Comment:
I see we still have the responseNode object around, so now there's two different clientConfigure response objects floating around.
If we can't completely remove responseNode I'm not sure what this change buys us ?

Reply:
Removed `responseNode` for `createClientPrefix`, but we still need to convert to JsonNode [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/45e705f2b56f6d3f0fb0adb8a71e84f2d8bac09f/src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestStage.java#L275).